src/app/dynamic/enterprise/theming/picker/style.scss

```diff
@@ -12,27 +12,20 @@
 @import 'variables';
 @import 'mixins';
 
-.km-account-settings-entry-label {
+.entry-label {
```

Comment:
More as a general question: 
Is it intended that there is no `km` prefix? 
And if yes: How are we deciding if we prefix it or not? Already saw the not-prefixed-version in some places, where it is in a `km-[...]` class, which makes sense.

Reply:
Yes, it is intended. There is no need to prefix values that are part of component-level stylesheets as they are encapsulated (with exception if we change it manually in the component config). However, when it comes to the global styles it is better to keep the prefix, especially the part that points to the component or view, i.e. `*-account-settings-*`. `km-*` in general is up to us, but at the moment it is used mostly in global rules. It can be changed later though once we decide if we want to keep it or not but it may be better to have it to avoid issues that may be hard to find.

Reply:
By the way, `*theme.scss` files that have theme mixins and are on component-level should be also considered global as they are referenced from global files, not from components themselfs.

Reply:
We've added them to keep the rules next to each other, thanks to this it's easier to find and modify the component look.

Reply:
Alright, sounds good 👍 Thanks 😄